packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -71,83 +70,89 @@ describe('HttpValidator', () => {
         });
       });
     });
-  });
 
-  describe('headers validation in enabled', () => {
-    const validate = (resourceExtension?: Partial<IHttpOperation>, length: number = 1) => () => {
-      expect(
-        validateInput({
-          resource: Object.assign(
-            {
-              method: 'get',
-              path: '/',
-              id: '1',
-              request: {},
-              responses: [{ code: '200' }],
-            },
-            resourceExtension,
-          ),
-          element: { method: 'get', url: { path: '/' } },
-        }),
-      ).toHaveLength(length);
-    };
-
-    describe('request is not set', () => {
-      it('validates headers', validate(undefined, 2));
-    });
-  });
+    describe('headers validation in enabled', () => {
+      const validate = (resourceExtension?: Partial<IHttpOperation>, length: number = 1) => () => {
+        expect(
+          validateInput({
+            resource: Object.assign(
+              {
+                method: 'get',
+                path: '/',
+                id: '1',
+                request: {},
+                responses: [{ code: '200' }],
+              },
+              resourceExtension,
+            ),
+            element: { method: 'get', url: { path: '/' } },
+          }),
+        ).toHaveLength(length);
```

Comment:
hm.. is this enough to say that `headers validation is enabled`? Yeah, I know it was already like this, but maybe we could save the snapshot or do something like `toBe`?

Reply:
Good point, effectively this test is a little bit flaky; however the validation is one of the pieces we'll refactor soon, so I am not going to bother so much at this point.